### ISSUE TITLE
LL-HLS: Live stream latency increases over time if the tab is in background or minimized and doesn't catch up

### ISSUE THREAD

**Issue by duggaraju** [shaka-project/shaka-player#4476]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
4.2.1

**Can you reproduce the issue with our latest release version?**
yes

**Can you reproduce the issue with the latest code from `main`?**
yes

**Are you using the demo app or your own custom app?**
both

**If custom app, can you reproduce the issue using our demo app?**

yes
**What browser and OS are you using?**
Windows with Edge/Chrome

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**

**What are the manifest and license server URIs?**

https://aka.ms/LowLatencyDemo.m3u8

**What configuration are you using?  What is the output of `player.getConfiguration()`?**

**What did you do?**

Load https://aka.ms/LowLatencyDemo.m3u8  and play

**What did you expect to happen?**
The player plays at the lowest latency possible.

**What actually happened?**

When the tab is in background or minimized the player falls behind but when the tab is maximized/brought to foreground, the player fully doesn't catch up and the latency increases over time.

**Comment by avelad** [comment]
@theodab, can you see if you can find the root cause? I've been looking but I can't find the error.

**Comment by theodab** [comment]
I tried repro'ing on a Mac for about an hour and saw no increase in latency.
I'll try Windows tomorrow.

**Comment by avelad** [comment]
@theodab I have been able to reproduce it on macOS with Chrome. Monday is a holiday in Spain, I will take you through the steps to reproduce the error on Tuesday.

**Comment by avelad** [comment]
@theodab, I have sent you the steps to reproduce the problem in our private chat.

**Comment by theodab** [comment]
@avelad I was able to repro it using your test page. Thanks, I didn't set those configuration values when testing initially.

I was also able to repro on nightly and v4.3.5 also, by using the same configuration values and the same asset. Oddly, though, it was a lot slower; your test page got 30-60s behind the live edge in under a minute of being minimized, but I had to leave the demo pages running for quite a while to get it to have similar losses. I wonder what the difference is.
